tests/dotless.Core.Test/Unit/Parser/ColorTests.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace dotless.Core.Test.Specs.Functions
+{
+    using NUnit.Framework;
+    using Color = Core.Parser.Tree.Color;
+
+    public class ColorTests
+    {
+        [Test]
+        public void TestColor()
```

Comment:
This name is a bit generic, considering the fix it's testing. Perhaps something like `Color_ConstructedFromHexString_RgbValuesDoNotThrow`?